src/roles/sdwan-portal-deploy/templates/config.j2

```diff
@@ -29,7 +29,9 @@ smtp.password: '{{ smtp_password }}'
 smtp.port: '{{ smtp_port | default('None') }}'
 smtp.secureConnection: {{ '1' if smtp_secure is defined and smtp_secure else '0' }}
 smtp.user: '{{ smtp_port | default('25') }}'
+{% if vstat_fqdn_global is defined %}
```

Comment:
Can you please check if it is NONE instead of defined. because group_vars.all.j2 defaults the variable to NONE to it will never be undefined

Reply:
done